app/dashboard/organization/history/nonProfitHistory.tsx

```diff
@@ -0,0 +1,276 @@
+                      {!nameFilter ? (
+                        <UnfoldMoreIcon fontSize="inherit" />
+                      ) : nameLeastToGreatest ? (
+                        <ExpandMoreIcon fontSize="inherit" />
+                      ) : (
+                        <ExpandLessIcon fontSize="inherit" />
+                      )}
+                    </button>
+                  </div>
+                  {/* Amount section */}
+                  <div className="basis-1/6 flex items-center space-x-2">
+                    <div>Amount</div>{" "}
+                    <button onClick={() => handleAmountSort()}>
+                      {!amountFilter ? (
+                        <UnfoldMoreIcon fontSize="inherit" />
+                      ) : amountLeastToGreatest ? (
+                        <ExpandMoreIcon fontSize="inherit" />
+                      ) : (
+                        <ExpandLessIcon fontSize="inherit" />
+                      )}
+                    </button>
+                  </div>
+                  {/* Org funded section */}
+                  <div className="basis-1/3">Grant Owner</div>
+                  {/* Date section */}
+                  <div className="basis-1/6 flex items-center space-x-2">
+                    <div>Date Accepted</div>
+                    <button onClick={() => handleDateSort()}>
+                      {!dateFilter ? (
+                        <UnfoldMoreIcon fontSize="inherit" />
+                      ) : dateLeastToGreatest ? (
+                        <ExpandMoreIcon fontSize="inherit" />
+                      ) : (
+                        <ExpandLessIcon fontSize="inherit" />
+                      )}
+                    </button>
+                  </div>
+                </div>
+                {/* End Heading */}
+
+                {/* Table Rows */}
+                <div className="2xl:min-h-[35rem] 2xl:max-h-[35rem] min-h-[25rem] max-h-[25rem] overflow-y-auto">
+                  {grantList.map((grant) => {
+                    return (
+                      <TableRow
+                        key={grant.id}
+                        id={grant.id}
+                        name={grant.name}
+                        amount={grant.amount}
+                        orgDonor={grant.ownerOrgName}
+                        date={grant.acceptedDate}
+                      />
+                    );
+                  })}
+                </div>
+              </div>
+
+              {/* Stats */}
+              <div className="flex justify-around 2xl:mt-16 mt-10">
+                {/* Total grants */}
+                <div className="text-center">
+                  <div className="text-5xl font-semibold text-maroon">
+                    {grantList.length}
+                  </div>
+                  <div className="text-lg mt-2 font-light">
+                    Total Grants Received
+                  </div>
+                </div>
+                {/* Total amount given */}
```

Comment:
Change comment to total amount received instead of given (for readability behind the scenes)